test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1788,6 +1788,23 @@ BOOST_AUTO_TEST_CASE(transfer_ether)
 	ABI_CHECK(callContractFunction("b(address,uint256)", oogRecipient, 10), encodeArgs());
 }
 
+BOOST_AUTO_TEST_CASE(uncalled_blockhash)
+{
+	char const* code = R"(
+		contract C {
+			function f() public view returns (bytes32)
+			{
+				var x = block.blockhash;
```

Comment:
There is no way to define a function type for this, because it has an "internal" type. This syntax is deprecated in the future so I guess we need to deal with this at that point.

Reply:
Exactly, the problem will go away later anyway.